config.template.yml

```diff
@@ -575,6 +580,12 @@ notifier:
     #   --- KEY START
     #   --- KEY END
 
+    ## The lifespans configure the expiration for these token types.
+    # access_token_lifespan: 1h
+    # authorize_code_lifespan: 1h
+    # id_token_lifespan: 1h
+    # refresh_token_lifespan: 720h
```

Comment:
We should probably advertise something smaller: https://docs.apigee.com/api-platform/antipatterns/oauth-long-expiration

Reply:
Yep I agree, currently the defaults are exactly what we have configured in 4.29.1 (by not configuring it). As I unwrap more of fosite I'm discovering more of these things.

Reply:
I adjusted this to 90m and documented it. Since people can tweak it there should be no issues. I also added a link to those docs.